tests/test_logs/test_logs.py

```diff
@@ -494,3 +492,48 @@ def test_describe_log_groups_paging():
     resp = client.describe_log_groups(nextToken="invalid-token")
     resp["logGroups"].should.have.length_of(0)
     resp.should_not.have.key("nextToken")
+
+
+@mock_logs
+def test_describe_log_streams_simple_paging():
```

Comment:
There was no testss at all for pagination in describe_log_streams(). I created the test first, and then switched to the global decorator to verify the implementation hasn't changed